protos/bottle/fulfillment/v1/line_item.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineItem {
+  string id = 1;
```

Comment:
Are we going to generate a ID for every LineItem?

Reply:
We already do for order product lines